posts/input-range.md

```diff
@@ -1,11 +1,11 @@
 feature: <input type=range>
-status: caution
-tags: polyfill gtie9
+status: use
+tags: gtie9
 kind: html
-polyfillurls: [Input Range Polyfill](https://github.com/freqdec/fd-slider), [rangeslider.js](http://andreruffert.github.io/rangeslider.js/), [html5slider](https://github.com/fryn/html5slider)
+polyfillurls: [Input Range Polyfill](https://github.com/freqdec/fd-slider), [rangeslider.js](http://andreruffert.github.io/rangeslider.js/)
 
-A range input (AKA slider) will fall back to a plain text input if it's not supported.
+A range input (AKA slider) is supported by all evergreen browsers and Internet Explorer 11. Also it will fall back to a plain text input if it's not supported.
 
-You can use freqdec's [Input Range Polyfill](https://github.com/freqdec/fd-slider), [rangeslider.js](http://andreruffert.github.io/rangeslider.js/). Or to support Firefox <= 22, try Frank Yan's [html5slider](https://github.com/fryn/html5slider).
+If you have to support for Internet Explorer 10 and below you can use freqdec's [Input Range Polyfill](https://github.com/freqdec/fd-slider), [rangeslider.js](http://andreruffert.github.io/rangeslider.js/).
```

Comment:
It’s supported by IE10 according to [can I use](https://caniuse.com/#search=input-range) plus its one superfluous word in this sentence.

So it should be 
```suggestion
If you have to support Internet Explorer 9 and below you can use freqdec's [Input Range Polyfill](https://github.com/freqdec/fd-slider), [rangeslider.js](http://andreruffert.github.io/rangeslider.js/).
```